src/components/pages/SingleEventPage.jsx

```diff
@@ -2,35 +2,90 @@ import { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useParams } from 'react-router-dom';
 import { fetchEventById } from '../../redux/events/operations_';
-import { getEvents } from '../../redux/selectors';
+import { getCurrentEvent, getLang } from '../../redux/selectors';
+import { Icon } from '@iconify/react';
+
 
 export const SingleEventPage = () => {
   const { id } = useParams();
   const dispatch = useDispatch();
+  const currentEvent = useSelector(getCurrentEvent);
+  const lang = useSelector(getLang);
 
   useEffect(() => {
     dispatch(fetchEventById(id));
   }, [dispatch, id]);
 
-  const { currentEvent_ } = useSelector(getEvents);
-  console.log(currentEvent_);
-
-  return currentEvent_ ? (
-    <>
-      <h1>
-        I am SingleEventPage of{' '}
-        <span className="text-red-600 font-bold">{currentEvent_.title.en}</span>
-      </h1>
-      <p className="text-red-600 font-bold">
-        Vadym, update me using 'currentEvent' redux state and this{' '}
-        <a
-          target="blank_"
-          href="https://www.eventbrite.ca/e/salsa-bachata-international-artist-weekender-sep-14-15-2024-tickets-892119351997?aff=erelexpmlt"
-          className="underline underline-offset-2"
-        >
-          template
-        </a>
-      </p>
-    </>
-  ) : null;
+  if (!currentEvent) {
+    return <div>Loading...</div>;
+  }
+
+  const {
+    coverImg,
+    title,
+    announce,
+    description,
+    startDate,
+    endDate,
+    address,
+    contacts,
+  } = currentEvent;
+
+  const startDateFormat = new Date(startDate).toLocaleDateString(lang === 'eng' ? 'en-EN' : 'ua-UA', {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  const endDateFormat = endDate
+    ? new Date(endDate).toLocaleDateString(lang === 'eng' ? 'en-EN' : 'ua-UA', {
+        weekday: 'long',
+        year: 'numeric',
+        month: 'long',
+        day: 'numeric',
+      })
+    : null;
+
+
+  return (
+    <div className="max-w-screen-xl mx-auto p-5 sm:p-10 md:p-16">
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-10">
+        <img src={coverImg} alt={title[lang]} className="rounded shadow-lg" />
+        <div>
+          <h1 className="text-3xl font-bold mb-3">{lang === 'eng' ? title.en : title.ua}</h1>
+          <p className="text-xl text-gray-700 mb-5">{announce[lang]}</p>
```

Comment:
{announce[lang]} - that is just perfect ! 
I should change all my ternary operators to this approach